server/plugin.go

```diff
@@ -390,3 +390,40 @@ func (p *Plugin) MessageWillBeUpdated(c *plugin.Context, newPost, oldPost *model
 
 	return newPost, ""
 }
+
+func (p *Plugin) UserHasLeftChannel(_ *plugin.Context, cm *model.ChannelMember, _ *model.User) {
+	if cm == nil {
+		p.LogWarn("UserHasLeftChannel: unexpected nil channel member")
+		return
+	}
+
+	state, err := p.kvGetChannelState(cm.ChannelId, false)
+	if err != nil {
+		p.LogError("UserHasLeftChannel: failed to get call state", "err", err.Error(), "channelID", cm.ChannelId)
+		return
+	} else if state == nil || state.Call == nil {
+		p.LogDebug("UserHasLeftChannel: no call ongoing", "channelID", cm.ChannelId)
```

Comment:
Do we need this? I'm worried it will spam the logs...

Reply:
That's the question, we need it when something suddenly breaks and we are clueless :)

I think that's what debug level logs are in my mind. It's a spam which you should be able to filter. It's more the times I had to debug something in production and wished I had these sorts of logs than the times I suffered through spam. But I agree with you, likely nothing too critical happening here so can probably be removed.